app/bundles/ProjectBundle/Service/ProjectEntityLoaderService.php

```diff
@@ -196,6 +208,7 @@ private function getEntityTypes(): array
                         entityClass: $entityClass,
                         label: $this->getEntityLabel($entityType),
                         model: $this->findModelForEntityType($entityType),
+                        detailRoute: $detailRoutes[$entityType] ?? new DetailRoute('mautic_'.$entityType.'_action', 'objectId', ['objectAction' => 'view']),
```

Comment:
```suggestion
                        detailRoute: $routeEvent->getRoute($entityType),
```
See my previous comment.